Stylesheets/common _css/general.css

```diff
@@ -0,0 +1,32 @@
+.container {
+    margin: 3rem auto;
+    width:clamp(15rem,20rem,25rem);
+    height:clamp(2.5rem,5rem,10rem);
+    background-color: var(--st-background-grey);
+    border: 2px solid grey;
+}
+
+.big-container{
```

Comment:
Give a generic name to class names.

Reply:
Hi Tanishq,
Can you please give example for the generic name to be used here.
Sorry I am a bit confused.